### DISCUSSION TITLE
[A-L] (2024/25) Foglio 1 - Esercizio 3

### DISCUSSION THREAD

**Discussion by Rurik-D** [sapienzastudentsnetwork/algebra#464] [Esercizi - Viaggi/Cherubini]
![image](https://github.com/user-attachments/assets/7705b084-3ff6-4b8b-849c-fadbda28a681)

**Comment by niccolo-fato** [comment]
Non so se sia corretto 
![photo_2024-09-30_11-54-46](https://github.com/user-attachments/assets/c4bc6cd8-2922-49a8-9da5-bcc7623fde70)
